Decred Wallet/Features/TransactionDetails/TransactionDetails.storyboard

```diff
@@ -18,67 +21,392 @@
+                                                            <outlet property="creditsStack" destination="pzz-MW-9F2" id="h3L-St-frz"/>
+                                                            <outlet property="headerButton" destination="jRZ-D6-Dst" id="MPy-MI-IMY"/>
+                                                        </connections>
+                                                    </tableViewCell>
+                                                    <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" preservesSuperviewLayoutMargins="YES" selectionStyle="none" indentationWidth="10" reuseIdentifier="TransactionViewOnDcrdataCell" id="pFD-j8-VAY">
+                                                        <rect key="frame" x="0.0" y="162" width="375" height="48"/>
+                                                        <autoresizingMask key="autoresizingMask"/>
+                                                        <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" preservesSuperviewLayoutMargins="YES" insetsLayoutMarginsFromSafeArea="NO" tableViewCell="pFD-j8-VAY" id="tGw-3D-4hO">
+                                                            <rect key="frame" x="0.0" y="0.0" width="375" height="48"/>
+                                                            <autoresizingMask key="autoresizingMask"/>
+                                                            <subviews>
+                                                                <button opaque="NO" clipsSubviews="YES" contentMode="scaleToFill" contentHorizontalAlignment="left" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="kOO-c0-kqG">
+                                                                    <rect key="frame" x="16" y="9" width="343" height="30"/>
+                                                                    <fontDescription key="fontDescription" name="SourceSansPro-Regular" family="Source Sans Pro" pointSize="14"/>
+                                                                    <state key="normal" title="View on dcrdata">
+                                                                        <color key="titleColor" red="0.035294117649999998" green="0.078431372550000003" blue="0.25098039220000001" alpha="1" colorSpace="calibratedRGB"/>
+                                                                    </state>
+                                                                    <userDefinedRuntimeAttributes>
+                                                                        <userDefinedRuntimeAttribute type="string" keyPath="xibLocalizedStringKey" value="viewOnDcrdata"/>
+                                                                    </userDefinedRuntimeAttributes>
+                                                                </button>
```

Comment:
Using a button for this `View on dcrdata` cell prevents the table view's didSelectRowAt method from firing when the button is tapped. Only gets fired if you tap the edges of the cell. Using a label fixes that.